workers/structure/tasks.py

```
@@ -0,0 +1,852 @@
+    logger.info(f"Fetching exported tool with UUID '{prompt_registry_id}'")
+
+    # Try to fetch as prompt studio tool first
+    tool_metadata = None
+    is_agentic = False
+    exported_tool = None
+
+    try:
+        exported_tool = platform_helper.get_prompt_studio_tool(
+            prompt_registry_id=prompt_registry_id
+        )
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)